src/db/models/Article.js

```diff
@@ -64,6 +60,7 @@ export default (sequelize, DataTypes) => {
     Article.belongsTo(models.User, {
       foreignKey: 'userId',
       as: 'author',
+      onDelete: 'CASCADE',
```

Comment:
I don't know if this works here in the model but I believe we set this in the migration already to cascade on delete